etna/core/mixins.py

```diff
@@ -89,6 +89,45 @@ def to_dict(self):
         params["_target_"] = BaseMixin._get_target_from_class(self)
         return params
 
+    def set_params(self, **params):
```

Comment:
I think we should add type hint here for `params`: https://stackoverflow.com/questions/37031928/type-annotations-for-args-and-kwargs

Reply:
We should probably add type hint for the return type.